src/components/AppBar/hooks.ts

```diff
@@ -0,0 +1,8 @@
+import { useDependencies } from "../../di";
+
+export const useLoginUrl = () => {
+  const container = useDependencies();
+  const apiUrl = container.resolve<string>("apiUrl");
+
+  return `${apiUrl}/authorize/google?redirect=http://localhost:3001/oauth2/callback`;
```

Comment:
How is this going to work not locally?

Reply:
Yep I forgot an envrionment variable. Its now fixed